cinema/urls.py

```diff
@@ -1 +1,22 @@
-# write urls here
+from django.urls import path, include
+from rest_framework import routers
+
+from cinema.views import (
+    ActorViewSet,
+    GenreViewSet,
+    CinemaHallViewSet,
+    MovieViewSet,
+    MovieSessionViewSet,
+)
+
+app_name = "cinema"
+
+router = routers.DefaultRouter()
+router.register("actors", ActorViewSet)
+router.register("genres", GenreViewSet)
+router.register("cinema_halls", CinemaHallViewSet)
+router.register("movies", MovieViewSet)
+router.register("movie_sessions", MovieSessionViewSet)
+
+
+urlpatterns = [path("", include(router.urls))]
```

Comment:
No need to include router when a basepath is not provided.
Use `urlpatterns = router.urls`